Constants.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Rsk.Saml.Samples;
+
+public static class Constants
+{
+    public const string Licensee = "Your licensee";
```

Comment:
Licensee shoudl be DEMO
and LicenseKey should be "visit https://www.identityserver.com/products/saml2p to obtain a free demo key"